### DISCUSSION TITLE
repo

### DISCUSSION THREAD

**Discussion by ghost** [community/community#66991] [Other Feature Feedback, Questions, & Ideas] [answered]

**Comment by ikramikram2020** [comment]
try to clone it

**Comment by Akash1134** [comment, marked as answer]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
